regression/strings-smoke-tests/java_format6/check_alternative.desc

```diff
@@ -0,0 +1,8 @@
+FUTURE
+test.class
+--refine-strings --string-max-length 20 --function test.check_alternative
+^EXIT=0$
+^SIGNAL=0$
+ASSERTION SUCCESSFUL
```

Comment:
I don't think `ASSERTION SUCCESSFUL` appears anywhere in the cbmc output. Please look at other (strings-smoke-) tests to see what line you should put there.